Repository/Vcs/GitDriver.php

```diff
@@ -35,4 +36,24 @@ public function getComposerInformation($identifier)
 
         return ProcessUtil::getComposerInformation($this->cache, $this->infoCache, $this->repoConfig['asset-type'], $this->process, $identifier, $resource, sprintf('git show %s', $resource), sprintf('git log -1 --format=%%at %s', escapeshellarg($identifier)), $this->repoDir, '@');
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function initialize()
+    {
+        if (isset($this->config->get('fxp-asset')['git-driver']['skip-update'])) {
+            $skip = $this->config->get('fxp-asset')['git-driver']['skip-update'];
+            if ($skip != 0) {
+                $localUrl = $this->config->get('cache-vcs-dir') . '/' . preg_replace('{[^a-z0-9.]}i', '-', $this->url) . '/';
```

Comment:
This is taken from https://github.com/composer/composer/blob/master/src/Composer/Repository/Vcs/GitDriver.php#L45

Why should we treat `_` differently and why change `-` at all?